projects/fw_102/src/main.c

```diff
@@ -0,0 +1,42 @@
+#include <stdint.h>  // for integer types
+
+#include "interrupt.h"   // interrupts are required for soft timers
```

Comment:
Good work with the comments, but the ones for the libraries are a bit overkill. If they are for your own reference in the homework that's totally fine, but not something we would do in production code 🙂